src/pages/MacKenzieAccessReview.jsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { div } from "react-hyperscript-helpers";
+import { Application } from '../components/Application';
+import { AccessReviewHeader } from '../components/AccessReviewHeader';
+import { VoteAsX } from '../components/VoteAsX';
+import { DAR } from '../libs/ajax';
+
+const SECTION = {
+  margin: '16px',
+};
+
+class MacKenzieAccessReview extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = this.initialState();
```

Comment:
I'd like to get away from this pattern of calling a method for initial state. I don't have a good handle on what I do want, so will look into that some more.